tests/Moq.Tests.FSharpTypes/HasActionEvent.fs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2007, Clarius Consulting, Manas Technology Solutions, InSTEDD.
+// All rights reserved. Licensed under the BSD 3-Clause License; see License.txt.
+
+namespace Moq.Tests.FSharpTypes
+
+open System;
+
+type HasActionEvent() =
+    let event = new DelegateEvent<Action>();
+
+    [<CLIEvent>]
+    member this.Event = event.Publish
```

Comment:
This type and event will probably be quite useless for other Moq unit tests, since `Event` and its accessor methods won't be defined `virtual` so Moq cannot intercept them.